client/screens/trip/createTrip.tsx

```diff
@@ -31,8 +31,10 @@ export default function Trips() {
     (state) => state.search.selectedSearchResult,
   );
 
-  const { latLng, selectedSearch } = useSelector((state) => state.weather);
-
+  const weather = useSelector((state) => state.weather);
```

Comment:
we are trying to move away from redux to trpc query, so we should update this with new logic